cmd/ocitools/validate.go

```diff
@@ -337,6 +337,17 @@ func checkLinux(spec rspec.Spec, rootfs string, hostCheck bool) (msgs []string)
 			} else if spec.Linux.Namespaces[index].Type == rspec.MountNamespace {
 				mountExists = true
 			}
+		} else {
+			if hostCheck {
```

Comment:
On Tue, Aug 16, 2016 at 08:43:42PM -0700, Ma Shimiao wrote:
> 08/17/2016 11:42 AM, W. Trevor King:
> > |} else if hostCheck {| to avoid a layer of nesting?
> I ever thought about this.
> But, it seems they should not in the seem level.

It doesn't really matter, so if it feels too strange go ahead and
leave it in there.  It just reads more clearly to me with fewer
levels.